src/index.tsx

```diff
@@ -2,12 +2,14 @@ import React from "react";
 import ReactDOM from "react-dom/client";
 import "./index.css";
 import App from "./App";
+import ForgotPassword from "./ForgotPassword";
 import reportWebVitals from "./reportWebVitals";
 
 const root = ReactDOM.createRoot(document.getElementById("root")!);
 root.render(
   <React.StrictMode>
-    <App />
+    {<ForgotPassword />}
+    {/*<App />*/}
```

Comment:
why is this being commented out?

Reply:
> why is this being commented out?

Just to test how ForgotPassword component renders in the browser as currently we have no routes set up. Changes made in this `src/index.tsx` file can be done away with.